README.md

```diff
@@ -16,10 +16,13 @@
 [Configuration & Docker Compose](https://hay-kot.github.io/homebox/quick-start)
 
 ```bash
-docker run --name=homebox \
-    --restart=always \
-    --publish=3100:7745 \
-    ghcr.io/hay-kot/homebox:latest
+docker run -d \
+  --name homebox \
+  --restart unless-stopped \
+  -p 3100:7745 \
+  -e TZ=Europe/Bucharest \
+  -v ${HOME}/.homebox:/data \
```

Comment:
Why do you think should it be a bind mount instead of a volume? 

--

I'm not sure I want to advocate people use their home directory over the named volume.

Given

https://github.com/hay-kot/homebox/blob/673db41f37714e0ae2d2beca1876d81a432bf8cc/Dockerfile#L47

the data should be persistent anyways

🤷

Reply:
you should mount the volume in a defined location for easy backups and access. home directory makes sense as there are the... user directories, configs and data. all backup solutions include the home directory by default.